CONTRIBUTING.md

```diff
@@ -1,269 +1,54 @@
-# Contributing to Exonum Core
+# Contributing to Exonum
 
-This contribution guide is partially derived from
-[Bitcoin Contribution Guide](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md)
+Exonum welcomes contribution from everyone in the form of [suggestions], [bug
+reports] or pull requests. This document gives some guidance if you are 
+thinking of helping us.
 
-The Exonum Core project and its separate modules operate an open contributor
-model where anyone is welcome to contribute towards development
-in the form of peer review, testing and patches.
-This document explains the practical process and guidelines for contributing.
-The guidelines are the same for other Exonum repositories.
+[Project overview] and [documentation] can help you to better understand what's
+going on here. 
```

Comment:
"what's going on here." may be it's better to make it more bookish like "current state of the project"